pkg/get/tools.go

```diff
@@ -647,6 +647,28 @@ https://github.com/inlets/inletsctl/releases/download/{{.Version}}/{{$fileName}}
 			{{.Name}}_{{$os}}_{{$arch}}.{{$extStr}}`,
 		})
 
+	tools = append(tools,
+		Tool{
+			Owner:       "aws",
+			Repo:        "eks-anywhere",
+			Name:        "eksctl-anywhere",
+			Description: "Run Amazon EKS on your own infrastructure",
+			BinaryTemplate: `
+			{{$os := .OS}}
+			{{$ext := "tar.gz"}}
+			{{$arch := .Arch}}
+			{{- if eq .Arch "x86_64" -}}
+			{{$arch = "amd64"}}
+			{{- else if eq .Arch "aarch64" -}}
+			{{$arch = "arm64"}}
+			{{- else if eq .Arch "arm64" -}}
+			{{$arch = "arm64"}}
```

Comment:
I don't believe you are using this code path, try deleting it and see if your unit tests still work